Assets/HoloToolkit/Input/Scripts/Cursor/AnimatedCursor.cs

```diff
@@ -69,20 +69,28 @@ public override void OnInputDisabled()
         /// Override to set the cursor animation trigger
         /// </summary>
         /// <param name="modifier"></param>
-        protected override void OnActiveModifier(CursorModifier modifier)
```

Comment:
Let's keep the old method and use it to call the method with the interface implementation and mark the old one obsolete.

Reply:
In line with the comment I have made above, if we keep the old method, it'll receive null as a parameter when gaze is on the game object with any other implementation of ICursorModifier(e.g., TestCursorModifier) and hence, the new method with the interface implementation will also get null.